docs/src/main/asciidoc/_elytron/components/Aggregate_Security_Realm.adoc

```diff
@@ -8,7 +8,9 @@ The `aggregate-realm` resource contains the following attributes: -
  * `authentication-realm` - This realm use used to load the credentials of the identity or perform evidence verification.
  * `authorization-realm` - The realm to use to load the identities attributes used for authorization.
  * `authorization-realms` - A list of one or more realms to use to load the identities attributes and aggregate into a single set of attributes.
- 
+ * `principal-transformer` - A principal transformer that can be used to transform the principal after loading the credentials
```

Comment:
Would be good to mention that this attribute is optional.

Reply:
Added